src/main/webapp/js/calendar.js

```diff
@@ -13,22 +13,187 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function initCalendar() {
-  new tui.Calendar('#calendar', {
-    defaultView: 'week',
-    useCreationPopup: true,
-    useDetailPopup: true,
-    disableDblClick: true,
-    disableClick: true,
-    isReadOnly: true,
-    scheduleView: ['time'],
-    taskView: false,
-    week: {
-      workweek: true,
-    },
+// TODO: Add ability to remove courses from the calendar
+// TODO: Add ability to choose which section the user wants, not just
+// automatically
+//  choosing the first section
```

Comment:
TODO's should contain an issue number or username, i.e.
`// TODO(savsa): Add ability to remove courses from the calendar`